app/src/main/res/layout/chapter_sort_bottom_sheet.xml

```diff
@@ -88,12 +88,26 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content" />
 
+            <com.google.android.material.button.MaterialButton
```

Comment:
I have no idea about the xml... I took the button from neko and made it so it appears lol